.circleci/config.yml

```diff
@@ -12,11 +12,25 @@ workflows:
         - test-2.071-prod
         - test-2.077-upstream-devel
         - test-2.077-upstream-prod
+        - d2tag:
+            context: org-global
+            requires:
+            - test-2.070-devel
+            - test-2.070-prod
+            - test-2.071-devel
+            - test-2.071-prod
+            - test-2.077-upstream-devel
+            - test-2.077-upstream-prod
+            filters:
+                tags:
+                    ignore: /.*\+d2$/
+                branches:
+                    ignore: /.*/
 
 templates:
     test: &test-template
         docker:
-        - image: sociomantictsunami/dlang:xenial-v3.2.0-rc.1
+        - image: sociomantictsunami/dlang:xenial-v3.2.0
```

Comment:
Ideally this should be changed in a separate commit (the changing of the tag, I'm not sure GitHub will display this comment in the right place :).